CHANGELOG.md

```diff
@@ -29,6 +29,12 @@ Pylance's copy of Pyright has been updated from 1.1.351 to 1.1.354.
 
 -   See Pyright's release notes for details: [1.1.352](https://github.com/microsoft/pyright/releases/tag/1.1.352), [1.1.353](https://github.com/microsoft/pyright/releases/tag/1.1.353), [1.1.354](https://github.com/microsoft/pyright/releases/tag/1.1.354)
 
+## 2024.3.2 (25 March 2024) Release
```

Comment:
Seems like it's in the wrong spot? Shouldn't it be by date released?

Reply:
Maybe not, then it looks like all those other changes are in this hotfix.

Reply:
We've been sorting by version number.